core/tests/release_sources/feconf.txt

```diff
@@ -19,3 +19,8 @@ CURRENT_COLLECTION_SCHEMA_VERSION = 4
 
 # The current version of story contents dict in the story schema.
 CURRENT_STORY_CONTENTS_SCHEMA_VERSION = 1
+
+INCOMING_EMAILS_DOMAIN_NAME = ''
+ADMIN_EMAIL_ADDRESS = 'admin@example.com'
+SYSTEM_EMAIL_ADDRESS = 'admin@example.com'
```

Comment:
Change to system@example.com, here and elsewhere, since using different email addresses makes the testing more robust.

Reply:
Done.